jira_scraper.py

```diff
@@ -58,6 +57,59 @@ def get_jira_data(
 
     return data["issues"]
 
+def assemble_jira_query(jira_projects: list = DEFAULT_JIRA_PROJECTS) -> str:
+    projects = " OR ".join([f"project={e}" for e in jira_projects])
```

Comment:
I would suggest we limit the DEFAULT_JIRA_PROJECTS to only 'OSPRH` and `OSPCIX` as all bugs are created in those.

Reply:
updating the MR.